bin/update_rust

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -euo pipefail
+
+COMMAND="rustup update"
+echo "> $COMMAND"
+eval "$COMMAND"
+echo ""
```

Comment:
No need for empty echo here, it's for creating a newline if the previous command did not print one before exiting. `rustup update` does print a newline so no need for this.

```suggestion
```